main/kotlin/com/example/people_managment_system/model/Company.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.people_managment_system.model
+
+import javax.persistence.*
+
+@Entity
+@Table(name = "companies")
+open class Company {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "company_id", nullable = false)
+    open var companyId: Int? = null

```

Comment:
Always think about the data and understand what we always need upon creating something and what can be null and added later.

In this case it seems `companyId` is vital to create this row in the table, so it should not be nullable.